tests/php-unit-tests/src/BaseTestCase/ItopTestCase.php

```diff
@@ -185,9 +185,7 @@ public static function GetAppRoot()
 			return APPROOT;
 		}
 
-		$sAppRootPath = static::GetFirstDirUpContainingFile(__DIR__, 'approot.inc.php');
-
-		return $sAppRootPath . '/';
+		return static::GetFirstDirUpContainingFile(__DIR__, 'approot.inc.php');
```

Comment:
Would be better like this, no ?
```suggestion
		return static::GetFirstDirUpContainingFile(__DIR__, 'approot.inc.php') . '/';
```

Reply:
No, that's why I changed the method, it will then again return `//` at the end..